package.json

```diff
@@ -1,31 +1,27 @@
 {
-  "name": "highlightjs-supercollider",
-  "version": "1.0.0",
-  "description": "highlight.js syntax definition for the Supercollider language",
-  "main": "supercollider.js",
+  "name": "highlightjs-sclang",
+  "version": "0.1.0",
+  "description": "sclang syntax highlighting plugin for highlight.js",
+  "main": "src/sclang.js",
   "scripts": {
-    "test": "jasmine"
+    "test": "mocha"
   },
-  "files": [
-    "supercollider.js"
-  ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/highlightjs/highlightjs-supercollider.git"
+    "url": "git+https://github.com/highlightjs/highlightjs-cypher.git"
```

Comment:
cypher?

Reply:
thx for the hint, forgot to change it - copied from cypher and transfered the existing supercollider source code then.